README.md

```diff
@@ -38,7 +38,6 @@ Hosting (production):
 ## External APIs
 
 - Twitch OAuth (application)
-- Twitch EventSub/Helix
```

Comment:
We still use the Helix API so we should keep that in.

Reply:
Apologies, I thought it was part of the same endpoint, not two separate things. Fixed now!